src/main/java/org/jenkins/plugins/statistics/gatherer/util/PropertyLoader.java

```diff
@@ -209,4 +209,21 @@ public static Boolean getShouldPublishToAwsSnsQueue() {
         String shouldPublishToAwsSnsQueueEnv = getEnvironmentProperty("statistics.endpoint.shouldPublishToAwsSnsQueue");
         return "true".equals(shouldPublishToAwsSnsQueueEnv);
     }
+
+    public static boolean getShouldSendToLogback() {
+        Boolean shouldSendToLogback = StatisticsConfiguration.get().getShouldSendToLogback();
+        if (shouldSendToLogback != null) {
+            return shouldSendToLogback;
+        }
+        String shouldSendToLogbackEnv = getEnvironmentProperty("statistics.endpoint.shouldSendToLogback");
+        return "true".equals(shouldSendToLogbackEnv);
```

Comment:
I'd refactor a bit here. I'd extract `"true"` to a constant. I'd do the same for the other function of this class that works like this. 

Also, is it possible that `shouldSendToLogbackEnv` is not all lowercase? Maybe convert it to lowercase just to be sure?

Reply:
True, however, I will add that check on the getEnvironmentProperty() implementation so that we can apply the same logic consistently to all the properties.

Same thing for the "true".equals(), will factor out into a isTrue()